### ISSUE TITLE
Document --legacy-watch/-L in --help

### ISSUE THREAD

**Issue by mfn** [remy/nodemon#1325]
- `nodemon -v`: 117.3
- `node -v`: v8.11.1
- Operating system/terminal environment: Ubuntu 14.04 in Vagrant VirtualBox
- Command you ran: `nodemon --verbose --watch src/ --legacy-watch`

I installed nodemon a few days ago and due to my setup, regular field change detection didn't work. I checked `--help` but there was nothing provided in that regard.

Only after I checked the github repo and went through the `README.md` I saw the `--legacy-watch` flag being mentioned.

I've used nodemon in the (way way) past where this flag wasn't necessary (1.4.something) so it didn't immediately occur to me.

TL;DR: I'm suggestion to please add --legacy-watch/-L to the `--help` output too, thanks 😄

**Comment by remy** [comment]
I presume you didn't read the full help? 

<img width="937" alt="screen shot 2018-04-30 at 12 32 57" src="https://user-images.githubusercontent.com/13700/39425308-a8c2dade-4c72-11e8-8043-5a775f4373d3.png">

The `--help` was getting overwhelming, so common options are in `--help`, and as per the output, in bold, is `--help options` where you'll find legacy.

**Comment by mfn** [comment]
I guess not, total #fail on my part. Sorry for the noise!
